ml_service/pipelines/diabetes_regression_build_train_pipeline.py

```diff
@@ -33,8 +33,7 @@ def main():
 
     # Create a reusable Azure ML environment
     environment = get_environment(
-        aml_workspace, e.aml_env_name, create_new=False)  # NOQA: E501
-
+        aml_workspace, e.aml_env_name, create_new=e.rebuild_env == "true")  # 
```

Comment:
Please remove the == "true" so it can just be whatever the actual value is

Reply:
Hmm I don't understand: the environment variables are not well-typed (i.e. they are strings). In that case the flag would evaluate to true all the time (as a non-empty string). Just like [here](https://github.com/microsoft/MLOpsPython/blob/e6dbfb10545905881e46e6d76855538ccd5add13/diabetes_regression/evaluate/evaluate_model.py#L122)

(same as above)

Reply:
That's right. It's confusing in Python. I would update env_variables.py to return a boolean (not string) value. E.g. return self._rebuild_env == "true"